projects/aws/image-builder/cmd/downloadartifacts.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"log"
+)
+
+var downloadArtifactsCmd = &cobra.Command{
+	Use:   "artifacts",
+	Short: "Downloads artifacts for airgapped builds",
+	Long:  "Downloads EKS-D and EKS-A artifacts used to build EKS-A Node images in airgapped environment",
+	Run: func(cmd *cobra.Command, args []string) {
+		// Validate Inputs
+		if bo.EKSAReleaseVersion == "" {
```

Comment:
Can we use this method in `init()` to validate the flag is provided?https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagRequired

Reply:
good catch. I dont know what I was thinking.